howdoi/howdoi.py

```diff
@@ -83,9 +85,14 @@ def get_proxies():
 
 def get_result(url):
     try:
-        return requests.get(url, headers={'User-Agent': random.choice(USER_AGENTS)}, proxies=get_proxies()).text
-    except requests.exceptions.SSLError as e:
-        print('[ERROR] Encountered an SSL Error. Try using HTTP instead of '
+        urllib2.install_opener(
+            urllib2.build_opener(
+                urllib2.ProxyHandler(get_proxies())
+            )
+        )
```

Comment:
What would be the advantage of using the urllib2 package here?  The requests packages help with readability and is cleaner.